src/api/types.ts

```diff
@@ -64,7 +63,7 @@ export interface TagRecord {
 
 export interface WriteTicketRecord {
   tag_list?: Array<number>;
-  assigned_user?: number;
+  assigned_user?: string;
```

Comment:
This is probably the source of confusion as assigned_user might appear to refer to a user record when in reality it refers to a member. The backend could be modified to be more clear. (leave it for now, but in the future let's clarify things in the backend.)

Reply:
fixed